cmd/copy.go

```diff
@@ -445,6 +445,11 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 		return cooked, err
 	}
 
+	// Since we already don't support preserving LMT on upload or S2S, it's fine to limit this to download.
+	if cooked.fromTo.IsDownload() && cooked.preserveSMBProperties && cooked.preserveLastModifiedTime {
+		return cooked, errors.New("SMB properties include a last modified time. Please prefer either the SMB property or the REST property")
```

Comment:
I'm not sure that users will know what this means.  Can we use the actual parameter names here, telling to use either param x or param y, but not both?

Reply:
Sure.